### DISCUSSION TITLE
Совместимость с плагинами и мини модами

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#87] [General]
meta для парсера
```D
META
{
	Parser = Game;
	Engine = G2A;
	Mod    = g2_classic.mod;
};
```

**Comment by Kor-Angar** [comment]
**(+)** Teleport out of the World https://steamcommunity.com/sharedfiles/filedetails/?id=2804969430

**Reply by Kor-Angar** [reply]
добавил пустую инстанцию для совместимости

[KA_TeleportTooFarAway_G2_Classic.zip](https://github.com/dosinabox/g2_classic/files/9276257/KA_TeleportTooFarAway_G2_Classic.zip)

**Comment by Kor-Angar** [comment]
(+) Sleep One Hour https://steamcommunity.com/sharedfiles/filedetails/?id=2813676501

**Comment by Kor-Angar** [comment]
(+) Дуэли с неименными Наёмниками https://steamcommunity.com/sharedfiles/filedetails/?id=2797152415

**Comment by Kor-Angar** [comment]
(+) Телепорт для НПС https://steamcommunity.com/sharedfiles/filedetails/?id=2803256461

**Comment by Kor-Angar** [comment]
(+) Items mission fix https://steamcommunity.com/sharedfiles/filedetails/?id=2841743939

**Reply by Kor-Angar** [reply]
[G2_ITEM_MISSION_FIX_B_GiveInvItems_Text_classic.zip](https://github.com/dosinabox/g2_classic/files/9276271/G2_ITEM_MISSION_FIX_B_GiveInvItems_Text_classic.zip)

**Comment by Kor-Angar** [comment]
(+)  "Оружие торговцев" + "Блокировка инвентаря" https://steamcommunity.com/sharedfiles/filedetails/?id=2802843642

**Reply by Kor-Angar** [reply]
[KA_Trader_NoClear_ItMw_G2_Classic.zip](https://github.com/dosinabox/g2_classic/files/9276296/KA_Trader_NoClear_ItMw_G2_Classic.zip)

**Comment by Kor-Angar** [comment]
Рандомный «конец» в диалогах у НПС. надо будет сделать. что там не хватает для счастья. но версия рандомного конца старая, а переписывать её сегодня обновляя сил нет, так как жарко. но когда сядет солнце надо не забыть это сделать.

**Reply by Kor-Angar** [reply]
потому что не ничего важнее рандомного конца :-)

**Reply by Kor-Angar** [reply]
опа. а я его же переделывал две недели назад :-) на память. версия от 25,07,2022
[_DIA_EXIT.zip](https://github.com/dosinabox/g2_classic/files/9276301/_DIA_EXIT.zip)

**Comment by Kor-Angar** [comment]
@dosinabox режет без ножа

![image](https://user-images.githubusercontent.com/49514764/198160395-128cfca6-9c01-457a-a925-5de9313f450f.png)

**Reply by dosinabox** [reply]
Были константы PRINT_NotEnoughLP и PRINT_NotEnoughLearnPoints с одинаковыми текстами. Лишняя была убрана, только вот в классике и в аддоне разные) Сегодня добавлю в классику PRINT_NotEnoughLP, чтобы было одинаково.
